src/PolygonClipper/PolygonClipper.cs

```diff
@@ -140,7 +140,12 @@ public Polygon Run()
         for (int i = 0; i < clipping.ContourCount; i++)
         {
             Contour contour = clipping[i];
-            contourId++;
+            bool exterior = operation != BooleanOperation.Difference;
```

Comment:
I added this by accident when attempting to debug the Rust. I don't think it's correct and shouldn't be required.